lib/git_service/branch.rb

```diff
@@ -7,12 +7,8 @@ def initialize(branch)
       @branch = branch
     end
 
-    def blob_at(path) # Rugged::Blob object for a given file path on this branch
-      blob_data = tip_tree.path(path)
-      blob = Rugged::Blob.lookup(rugged_repo, blob_data[:oid])
-      (blob.type == :blob) ? blob : nil
-    rescue Rugged::TreeError
-      nil
+    def content_at(path)
+      blob_at(path).try(:content).try(:to_s)
```

Comment:
Good call...I only added it because the only caller we had it, but yeah doesn't look necessary.